src/board/v2/platform_signal.c

```diff
@@ -25,3 +27,10 @@ void platform_decode_setup(void)
 {
   decode_gps_l1ca_register();
 }
+
+void platform_ndb_init(void)
+{
+  ndb_init();
```

Comment:
Track and decode do:
- <module>_init() from main.c
- platform_<module>_init() from <module>_init()